orca/orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/tasks/WebhookResponseProcessor.java

```diff
@@ -190,7 +192,7 @@ private TaskResult processResponse(ResponseEntity response) {
     if (!response.getHeaders().isEmpty()) {
       webhookOutput.setHeaders(response.getHeaders().toSingleValueMap());
     }
-    HttpStatus status = response.getStatusCode();
+    HttpStatus status = HttpStatus.valueOf(response.getStatusCode().value());
```

Comment:
```suggestion
    HttpStatus status = response.getStatusCode();
```
works for me locally.